🐛(website) fix disconnection when multiple tabs (#2119)
openfun/marsha: CHANGELOG.md

## [Unreleased]

### Fixed

> - Bug disconnection when multiple tabs were open on standalone site

Reviewer: ```suggestion
- Manage disconnection when multiple tabs are open on standalone site
```
